pages/pipelines/build_matrix.md.erb

```diff
@@ -0,0 +1,126 @@
+# Build Matrix
+
+Build Matrices lets you simplify complex build configurations by expanding a step template and array of matrix elements into multiple jobs.
+
+{:toc}
+
+For example, instead of writing three separate jobs for builds on **macOS**, **Linux** and **Windows**, like the following build configuration (which does not use a build matrix):
```

Comment:
```suggestion
For example, instead of writing three separate jobs for builds on **macOS**, **Linux**, and **Windows**, like in the following build configuration (which does not use a Build Matrix):
```